### PR TITLE
Contributing...

### PR DESCRIPTION
**Pull request by cypher-adi**
Modern
Added a centered introduction with the dark theme to match the modern Cover.

### CODE REVIEW

File: src/modules/resume/introduction/variants/modern/render.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        backgroundColor: theme.value.color,
+        color: theme.value.contrast
+      }}
+    >
+      <hr />
+      <h1 style={styles.headlineText}>{headlineText.value}</h1>
+      <hr />
+      {paragraphs.value.map((paragraph) => {
+        return (
+          <div>
```
Comment (RisingGeek) [inline src/modules/resume/introduction/variants/modern/render.js:31]: Add key prop

**Comment by AKAspanion** [comment]
@cypher-adi Add a screenshot in the description?

**Comment by cypher-adi** [comment]
Modern

**Comment by cypher-adi** [comment]
Done sir.

File: src/modules/resume/introduction/variants/modern/render.js (hunk 1 above)
Comment (cypher-adi) [reply src/modules/resume/introduction/variants/modern/render.js:31]:
```suggestion
          <div key={paragraph.name}>
```

**Comment by ManojSatishkumar** [comment]
Can the basic introduction component have a property to align text to center instead of a brand new component ?

**Comment by cypher-adi** [comment]
@ManojSatishkumar Yes it can for sure but I did came up with that dark purple color to match with the Modern (dark) cover.

**Comment by ManojSatishkumar** [comment]
The color is adjustable on the editor when the component goes live. Check mskdocuments.com

Would be good if you enhance the basic variant and add your details to the contributors array.

**Comment by cypher-adi** [comment]
Sure sir will do that and come up with a PR.

**Comment by cypher-adi** [comment]
Enhanced the basic variant with center aligned text.

**Comment by cypher-adi** [comment]
Added a minimal skill-set variant with theme color bottom border
skillset

**Comment by ManojSatishkumar** [comment]
Can you create a separate branch and send PR with the branch to master ?

**Comment by ManojSatishkumar** [comment]
You should not change other's code.
You need to add a new property in Introductions component and read that to change text alignment to center

**Comment by ManojSatishkumar** [comment]
Please think about different designs and come back with new PR.

**Pull request closed without merging**